editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -143,11 +180,20 @@ function Component(props: Props) {
               previouslySubmittedData?._address?.source === "proposed" &&
               previouslySubmittedData?._address
             }
-            boundary={boundary}
             id={props.id}
             description={props.newAddressDescription || ""}
             descriptionLabel={props.newAddressDescriptionLabel || ""}
+            mapValidationError={mapValidationError}
+            setMapValidationError={setMapValidationError}
+            showSiteDescriptionError={showSiteDescriptionError}
+            setShowSiteDescriptionError={setShowSiteDescriptionError}
           />
+          {Boolean(address) && isValidating && (
+            <DelayedLoadingIndicator
+              msDelayBeforeVisible={50}
+              text="Fetching data..."
+            />
```

Comment:
_Either_ page needs to wait to fetch supplmentary address data, but we were incorrectly only showing the loading indicator on the select-existing-address page before. Now we'll show it on both pages (this impacts this draft PR #3136).

Currently, the overall `Card` will disable the continue button when `isValidating === true` - I'll address this last.